docs/src/content/docs/api/calls.mdx

```diff
@@ -537,6 +542,83 @@ Call recordings are only generated if you add `"recordingEnabled": true` to the
+
+* **Messages are Ordered** → Messages are delivered by the agent in the order provided.
+* **Messages are Cumulative** → Each duration is added to prior message durations.
+* **User Interactions Reset** → Any activity from the user will reset the message sequence.
+* **Different Behaviors** → Messages can have different end behaviors and can terminate the call.
+
+##### Best Practices
+* Keep messages concise and natural-sounding.
+* Start with friendly check-in messages before moving to call termination.
+* Provide clear context in messages if the call will be terminated.
+
+
+##### Message Format
+When creating a new call, `inactivityMessages` are an array of message objects. Each message must provide the following:
+
+<dl class="space-y-4">
+  <div>
+    <dt class="font-mono">duration</dt>
+    <dd class="ml-4"><span class="text-gray-600">string</span> - The duration (in seconds) after which the message should be spoken.</dd>
+    <dd class="ml-4"><span class="text-gray-600">pattern</span> - `^-?(?:0|[1-9][0-9]{0,11})(?:\.[0-9]{1,9})?s$`</dd>
+    <dd class="ml-4"><span class="text-gray-600">examples</span> - "60s", "5.5s"</dd>
+  </div>
+  <div>
+    <dt class="font-mono">message</dt>
+    <dd class="ml-4"><span class="text-gray-600">string</span> - The message to speak.</dd>
+    <dd class="ml-4"><span class="text-gray-600">examples</span> - "Are you still there?", "If there's nothing else, I will end the call now."</dd>
+  </div>
+  <div>
+    <dt class="font-mono">endBehavior</dt>
+    <dd class="ml-4"><span class="text-gray-600">string</span> - The behavior to exhibit when the message is finished being spoken. Must be one of the enumerated values.</dd>
+    <dd class="ml-4"><span class="text-gray-600">enum</span></dd>
+    <dd class="ml-8">
+      <ul class="list-disc space-y-2">
+        <li><code>END_BEHAVIOR_UNSPECIFIED</code> - Default. The system will continue to wait for user input.</li>
+        <li><code>END_BEHAVIOR_HANG_UP_SOFT</code> - Will end the call unless the user interacts while the agent is delivering the message.</li>
+        <li><code>END_BEHAVIOR_HANG_UP_STRICT</code> - Immediately terminates the call after speaking the message, without waiting for any additional user interaction.</li>
+      </ul>
+    </dd>
+  </div>
+</dl>
+
+##### Example
+
+```js
+{
+  "systemPrompt": "You are a helpful assistant.",
+  "inactivityMessages": [
+    {
+      "duration": "5s",
+      "message": "Hello, are you still there?"
+    },
+    {
+      "duration": "10s",
+      "message": "If there's nothing else, may I end the call?"
+    },
+    {
+      "duration": "5s",
+      "message": "Thank you for calling. Have a great day. Goodbye.",
+      "endBehavior": "END_BEHAVIOR_HANG_UP_STRICT"
```

Comment:
These durations are too tight for an example. Also non-strict hangups are probably preferable in most cases.

```suggestion
      "duration": "30s",
      "message": "Are you still there?"
    },
    {
      "duration": "30s",
      "message": "If there's nothing else, may I end the call?"
    },
    {
      "duration": "45s",
      "message": "Thank you for calling. Have a great day. Goodbye.",
      "endBehavior": "END_BEHAVIOR_HANG_UP_SOFT"
```

Reply:
committing this but will edit durations a bit